src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -33,27 +33,28 @@ class AptitudePackageManager(PackageManager):
     def __init__(self, env_layer, execution_config, composite_logger, telemetry_writer, status_handler):
         super(AptitudePackageManager, self).__init__(env_layer, execution_config, composite_logger, telemetry_writer, status_handler)
 
-        security_list_guid = str(uuid.uuid4())
-
         # Accept EULA (End User License Agreement) as per the EULA settings set by user
         optional_accept_eula_in_cmd = "ACCEPT_EULA=Y" if execution_config.accept_package_eula else ""
 
         # Repo refresh
-        self.repo_refresh = 'sudo apt-get -q update'
+        self.cmd_repo_refresh_template = 'sudo apt-get -q update <SOURCES>'
 
         # Support to get updates and their dependencies
-        self.security_sources_list = os.path.join(execution_config.temp_folder, 'msft-patch-security-{0}.list'.format(security_list_guid))
-        self.prep_security_sources_list_cmd = 'sudo grep -hR security /etc/apt/sources.list /etc/apt/sources.list.d/ > ' + os.path.normpath(self.security_sources_list)
-        self.dist_upgrade_simulation_cmd_template = 'LANG=en_US.UTF8 sudo apt-get -s dist-upgrade <SOURCES> '  # Dist-upgrade simulation template - <SOURCES> needs to be replaced before use; sudo is used as sometimes the sources list needs sudo to be readable
-        self.single_package_check_versions = 'apt-cache madison <PACKAGE-NAME>'
-        self.single_package_find_installed_dpkg = 'sudo dpkg -s <PACKAGE-NAME>'
-        self.single_package_find_installed_apt = 'sudo apt list --installed <PACKAGE-NAME>'
-        self.single_package_upgrade_simulation_cmd = '''DEBIAN_FRONTEND=noninteractive ''' + optional_accept_eula_in_cmd + ''' apt-get -y --only-upgrade true -s install '''
+        self.cached_customer_source_list_formula = None
+        self.custom_sources_list = os.path.join(execution_config.temp_folder, 'azgps-patch-custom-{0}.list'.format(str(uuid.uuid4())))
+        self.cmd_prep_custom_sources_list_template = 'sudo grep -hR <CLASSIFICATION> /etc/apt/sources.list /etc/apt/sources.list.d/ > ' + os.path.normpath(self.custom_sources_list)
+        self.cmd_dist_upgrade_simulation_template = 'LANG=en_US.UTF8 sudo apt-get -s dist-upgrade <SOURCES> '  # Dist-upgrade simulation template - <SOURCES> needs to be replaced before use; sudo is used as sometimes the sources list needs sudo to be readable
+
+        self.cmd_single_package_check_versions_template = 'apt-cache madison <PACKAGE-NAME>'
+        self.cmd_single_package_find_install_dpkg_template = 'sudo dpkg -s <PACKAGE-NAME>'
+        self.cmd_single_package_find_install_apt_template = 'sudo apt list --installed <PACKAGE-NAME>'
+        self.single_package_upgrade_simulation_cmd = '''DEBIAN_FRONTEND=noninteractive ''' + optional_accept_eula_in_cmd + ''' LANG=en_US.UTF8 apt-get -y --only-upgrade true -s install '''
```

Comment:
nit: naming format differs here, others start with cmd*, this one ends with it, Also do we need to end this with *template as well?

cmd* q for single_package_dependency_resolution_template in the line below as well

Reply:
cmd* q for lines 56 and 57 as well

Reply:
I didn't change this because it is correlated between package managers. Left it out to not add complexity in prod and test code in this reduced PR.